debian/changelog

```diff
@@ -1,3 +1,11 @@
+cloud-init (24.4~3+really24.3.1g337c6514-0ubuntu1) oracular; urgency=medium
```

Comment:
Note the version here contains `+really24.3.1`. This is due to my previous release to oracular accidentally bumping the release major.minor to 24.4 instead of 24.3. the `+really` is considered best practice in cases where you actually want to represent an upstream version in the packaging, but the current deb package already surpassed that version for one reason or another. This gives us a breadcrumb in the version which points to valid upstream release even though we can't actually use `24.3*`  in oracular.

This version validates as greater than the previous released version to oracular. We want to retain the initial `24.4~3` to ensure we can also release a `24.3~4` if we have other fixes during feature freeze before Oracular publishes.
```
dpkg --compare-versions 24.4~3+really24.3.1g337c6514-0ubuntu1 gt 24.4~2g2e4c39b7-0ubuntu1 && echo "Correct upgrade path"
```